[2024-06-06T17:39:15.695+0000] {processor.py:161} INFO - Started process (PID=171) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:15.696+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:39:15.700+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:15.699+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:15.773+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:15.750+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/main.py", line 3, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2024-06-06T17:39:15.779+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:15.803+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.114 seconds
[2024-06-06T17:39:25.957+0000] {processor.py:161} INFO - Started process (PID=176) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:25.958+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:39:25.961+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:25.960+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:26.032+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:26.018+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/main.py", line 3, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2024-06-06T17:39:26.037+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:26.063+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.108 seconds
[2024-06-06T17:39:36.191+0000] {processor.py:161} INFO - Started process (PID=181) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:36.193+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:39:36.196+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:36.195+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:36.253+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:36.240+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/main.py", line 3, in <module>
    import spotipy
ModuleNotFoundError: No module named 'spotipy'
[2024-06-06T17:39:36.258+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:36.284+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.098 seconds
[2024-06-06T17:39:46.498+0000] {processor.py:161} INFO - Started process (PID=186) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:46.501+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:39:46.535+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:46.535+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:46.674+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:46.658+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:39:46.680+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:46.742+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.285 seconds
[2024-06-06T17:39:56.889+0000] {processor.py:161} INFO - Started process (PID=191) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:56.891+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:39:56.893+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:56.892+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:56.948+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:39:56.942+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:39:56.951+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:39:56.975+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.091 seconds
[2024-06-06T17:40:07.110+0000] {processor.py:161} INFO - Started process (PID=196) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:07.112+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:40:07.118+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:07.116+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:07.175+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:07.165+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:40:07.179+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:07.214+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.109 seconds
[2024-06-06T17:40:17.360+0000] {processor.py:161} INFO - Started process (PID=201) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:17.362+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:40:17.365+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:17.365+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:17.404+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:17.397+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:40:17.406+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:17.428+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.077 seconds
[2024-06-06T17:40:27.567+0000] {processor.py:161} INFO - Started process (PID=206) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:27.569+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:40:27.572+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:27.572+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:27.616+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:27.609+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:40:27.619+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:27.648+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.088 seconds
[2024-06-06T17:40:37.809+0000] {processor.py:161} INFO - Started process (PID=211) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:37.812+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:40:37.815+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:37.815+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:37.871+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:37.862+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:40:37.874+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:37.922+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.125 seconds
[2024-06-06T17:40:48.084+0000] {processor.py:161} INFO - Started process (PID=216) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:48.086+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:40:48.089+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:48.088+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:48.150+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:48.138+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:40:48.155+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:48.191+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.115 seconds
[2024-06-06T17:40:58.354+0000] {processor.py:161} INFO - Started process (PID=221) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:58.356+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:40:58.358+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:58.358+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:58.417+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:40:58.408+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:40:58.420+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:40:58.450+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.105 seconds
[2024-06-06T17:41:08.607+0000] {processor.py:161} INFO - Started process (PID=226) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:08.609+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:41:08.611+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:08.611+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:08.686+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:08.674+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:41:08.692+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:08.732+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.136 seconds
[2024-06-06T17:41:18.864+0000] {processor.py:161} INFO - Started process (PID=231) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:18.865+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:41:18.869+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:18.868+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:18.914+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:18.907+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:41:18.917+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:18.944+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.086 seconds
[2024-06-06T17:41:29.077+0000] {processor.py:161} INFO - Started process (PID=236) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:29.078+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:41:29.080+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:29.080+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:29.118+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:29.112+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:41:29.120+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:29.142+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.071 seconds
[2024-06-06T17:41:39.264+0000] {processor.py:161} INFO - Started process (PID=241) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:39.266+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:41:39.268+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:39.267+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:39.309+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:39.302+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:41:39.312+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:39.334+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.074 seconds
[2024-06-06T17:41:49.488+0000] {processor.py:161} INFO - Started process (PID=246) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:49.490+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:41:49.492+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:49.492+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:49.545+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:49.536+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:41:49.548+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:49.572+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.093 seconds
[2024-06-06T17:41:59.760+0000] {processor.py:161} INFO - Started process (PID=251) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:59.761+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:41:59.765+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:59.764+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:59.830+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:41:59.819+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:41:59.835+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:41:59.866+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.117 seconds
[2024-06-06T17:42:10.012+0000] {processor.py:161} INFO - Started process (PID=256) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:10.013+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:42:10.015+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:10.015+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:10.054+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:10.048+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:42:10.056+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:10.079+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.073 seconds
[2024-06-06T17:42:20.208+0000] {processor.py:161} INFO - Started process (PID=261) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:20.211+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:42:20.217+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:20.216+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:20.273+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:20.265+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:42:20.277+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:20.331+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.134 seconds
[2024-06-06T17:42:30.509+0000] {processor.py:161} INFO - Started process (PID=266) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:30.510+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:42:30.513+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:30.512+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:30.569+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:30.560+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:42:30.574+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:30.604+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.103 seconds
[2024-06-06T17:42:40.838+0000] {processor.py:161} INFO - Started process (PID=271) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:40.841+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:42:40.845+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:40.844+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:40.909+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:40.900+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:42:40.913+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:40.952+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.126 seconds
[2024-06-06T17:42:51.210+0000] {processor.py:161} INFO - Started process (PID=276) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:51.213+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:42:51.219+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:51.218+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:51.307+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:42:51.296+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:42:51.312+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:42:51.357+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.160 seconds
[2024-06-06T17:43:01.585+0000] {processor.py:161} INFO - Started process (PID=281) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:01.587+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:43:01.591+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:01.591+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:01.657+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:01.646+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:43:01.662+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:01.693+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.115 seconds
[2024-06-06T17:43:11.897+0000] {processor.py:161} INFO - Started process (PID=286) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:11.900+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:43:11.904+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:11.903+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:11.999+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:11.985+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:43:12.004+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:12.054+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.166 seconds
[2024-06-06T17:43:22.235+0000] {processor.py:161} INFO - Started process (PID=291) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:22.237+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:43:22.240+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:22.240+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:22.295+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:22.287+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:43:22.299+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:22.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.100 seconds
[2024-06-06T17:43:32.508+0000] {processor.py:161} INFO - Started process (PID=296) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:32.511+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:43:32.516+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:32.515+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:32.575+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:32.566+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:43:32.580+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:32.615+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.115 seconds
[2024-06-06T17:43:42.823+0000] {processor.py:161} INFO - Started process (PID=301) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:42.824+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:43:42.827+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:42.827+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:42.877+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:42.869+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:43:42.880+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:42.909+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.095 seconds
[2024-06-06T17:43:53.170+0000] {processor.py:161} INFO - Started process (PID=306) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:53.173+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:43:53.177+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:53.177+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:53.225+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:43:53.217+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:43:53.229+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:43:53.281+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.135 seconds
[2024-06-06T17:44:03.524+0000] {processor.py:161} INFO - Started process (PID=311) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:03.527+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:44:03.531+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:03.531+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:03.596+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:03.584+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:44:03.601+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:03.638+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.125 seconds
[2024-06-06T17:44:13.837+0000] {processor.py:161} INFO - Started process (PID=316) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:13.839+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:44:13.843+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:13.842+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:13.907+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:13.896+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:44:13.912+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:13.951+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.126 seconds
[2024-06-06T17:44:24.210+0000] {processor.py:161} INFO - Started process (PID=321) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:24.212+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:44:24.217+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:24.216+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:24.269+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:24.260+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:44:24.273+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:24.300+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.097 seconds
[2024-06-06T17:44:34.602+0000] {processor.py:161} INFO - Started process (PID=326) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:34.605+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:44:34.611+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:34.610+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:34.687+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:34.675+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:44:34.693+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:34.738+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.148 seconds
[2024-06-06T17:44:44.996+0000] {processor.py:161} INFO - Started process (PID=331) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:44.998+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:44:45.002+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:45.001+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:45.064+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:45.052+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:44:45.070+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:45.109+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.122 seconds
[2024-06-06T17:44:55.357+0000] {processor.py:161} INFO - Started process (PID=336) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:55.360+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:44:55.367+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:55.366+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:55.456+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:44:55.446+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:44:55.461+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:44:55.501+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.157 seconds
[2024-06-06T17:45:05.698+0000] {processor.py:161} INFO - Started process (PID=341) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:05.700+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:45:05.704+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:05.703+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:05.763+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:05.754+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:45:05.766+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:05.802+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.111 seconds
[2024-06-06T17:45:16.055+0000] {processor.py:161} INFO - Started process (PID=346) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:16.058+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:45:16.064+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:16.063+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:16.142+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:16.129+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:45:16.147+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:16.192+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.153 seconds
[2024-06-06T17:45:26.368+0000] {processor.py:161} INFO - Started process (PID=351) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:26.370+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:45:26.372+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:26.372+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:26.428+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:26.417+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:45:26.432+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:26.463+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.101 seconds
[2024-06-06T17:45:36.707+0000] {processor.py:161} INFO - Started process (PID=356) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:36.710+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:45:36.715+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:36.714+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:36.779+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:36.768+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:45:36.783+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:36.815+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.120 seconds
[2024-06-06T17:45:47.001+0000] {processor.py:161} INFO - Started process (PID=361) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:47.003+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:45:47.005+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:47.005+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:47.059+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:47.050+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:45:47.063+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:47.091+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.100 seconds
[2024-06-06T17:45:57.261+0000] {processor.py:161} INFO - Started process (PID=366) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:57.262+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:45:57.265+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:57.264+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:57.315+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:45:57.307+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:45:57.319+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:45:57.349+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.096 seconds
[2024-06-06T17:46:07.553+0000] {processor.py:161} INFO - Started process (PID=371) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:07.555+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:46:07.559+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:07.558+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:07.631+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:07.621+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:46:07.636+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:07.673+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.131 seconds
[2024-06-06T17:46:17.923+0000] {processor.py:161} INFO - Started process (PID=376) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:17.932+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:46:17.941+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:17.940+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:18.038+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:18.023+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:46:18.045+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:18.098+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.194 seconds
[2024-06-06T17:46:28.292+0000] {processor.py:161} INFO - Started process (PID=381) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:28.294+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:46:28.297+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:28.297+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:28.344+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:28.335+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:46:28.348+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:28.388+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.103 seconds
[2024-06-06T17:46:38.614+0000] {processor.py:161} INFO - Started process (PID=386) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:38.616+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:46:38.620+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:38.619+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:38.701+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:38.687+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:46:38.707+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:38.755+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.149 seconds
[2024-06-06T17:46:49.018+0000] {processor.py:161} INFO - Started process (PID=391) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:49.021+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:46:49.027+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:49.026+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:49.133+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:49.119+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:46:49.141+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:49.194+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.189 seconds
[2024-06-06T17:46:59.408+0000] {processor.py:161} INFO - Started process (PID=396) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:59.411+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:46:59.414+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:59.414+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:59.484+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:46:59.470+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:46:59.489+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:46:59.525+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.126 seconds
[2024-06-06T17:47:09.767+0000] {processor.py:161} INFO - Started process (PID=401) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:09.770+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:47:09.775+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:09.774+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:09.857+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:09.843+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:47:09.863+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:09.912+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.159 seconds
[2024-06-06T17:47:20.144+0000] {processor.py:161} INFO - Started process (PID=406) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:20.146+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:47:20.152+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:20.151+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:20.219+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:20.207+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:47:20.223+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:20.260+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.125 seconds
[2024-06-06T17:47:30.496+0000] {processor.py:161} INFO - Started process (PID=411) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:30.499+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:47:30.505+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:30.503+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:30.575+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:30.564+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:47:30.581+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:30.620+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.136 seconds
[2024-06-06T17:47:40.839+0000] {processor.py:161} INFO - Started process (PID=416) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:40.841+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:47:40.845+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:40.845+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:40.907+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:40.898+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:47:40.912+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:40.946+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.116 seconds
[2024-06-06T17:47:51.156+0000] {processor.py:161} INFO - Started process (PID=421) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:51.158+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:47:51.163+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:51.162+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:51.255+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:47:51.243+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:47:51.261+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:47:51.299+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.152 seconds
[2024-06-06T17:48:01.556+0000] {processor.py:161} INFO - Started process (PID=426) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:01.559+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:48:01.565+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:01.564+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:01.633+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:01.624+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:48:01.638+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:01.678+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.135 seconds
[2024-06-06T17:48:11.921+0000] {processor.py:161} INFO - Started process (PID=431) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:11.923+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:48:11.927+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:11.926+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:11.997+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:11.988+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:48:12.000+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:12.037+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.128 seconds
[2024-06-06T17:48:22.233+0000] {processor.py:161} INFO - Started process (PID=436) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:22.235+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:48:22.240+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:22.239+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:22.314+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:22.302+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:48:22.320+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:22.359+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.136 seconds
[2024-06-06T17:48:32.564+0000] {processor.py:161} INFO - Started process (PID=441) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:32.567+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:48:32.571+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:32.571+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:32.689+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:32.646+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:48:32.692+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:32.719+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.166 seconds
[2024-06-06T17:48:42.917+0000] {processor.py:161} INFO - Started process (PID=446) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:42.920+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:48:42.925+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:42.924+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:43.019+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:43.006+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:48:43.027+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:43.085+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.186 seconds
[2024-06-06T17:48:53.275+0000] {processor.py:161} INFO - Started process (PID=451) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:53.276+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:48:53.281+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:53.280+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:53.344+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:48:53.333+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:48:53.349+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:48:53.385+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.120 seconds
[2024-06-06T17:49:03.572+0000] {processor.py:161} INFO - Started process (PID=456) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:03.574+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:49:03.578+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:03.577+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:03.625+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:03.616+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:49:03.629+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:03.657+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.092 seconds
[2024-06-06T17:49:13.867+0000] {processor.py:161} INFO - Started process (PID=461) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:13.869+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:49:13.873+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:13.872+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:13.959+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:13.945+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:49:13.965+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:14.010+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.155 seconds
[2024-06-06T17:49:24.168+0000] {processor.py:161} INFO - Started process (PID=466) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:24.171+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:49:24.175+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:24.174+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:24.242+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:24.233+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:49:24.247+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:24.273+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.112 seconds
[2024-06-06T17:49:34.492+0000] {processor.py:161} INFO - Started process (PID=471) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:34.494+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:49:34.499+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:34.498+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:34.566+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:34.554+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:49:34.572+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:34.605+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.121 seconds
[2024-06-06T17:49:44.826+0000] {processor.py:161} INFO - Started process (PID=476) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:44.829+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:49:44.834+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:44.833+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:44.928+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:44.914+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:49:44.935+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:44.992+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.180 seconds
[2024-06-06T17:49:55.217+0000] {processor.py:161} INFO - Started process (PID=481) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:55.219+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:49:55.222+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:55.222+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:55.278+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:49:55.269+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:49:55.283+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:49:55.324+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.116 seconds
[2024-06-06T17:50:05.526+0000] {processor.py:161} INFO - Started process (PID=486) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:05.527+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:50:05.533+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:05.533+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:05.608+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:05.595+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7, in <module>
    from main import main  # Importar la función principal de main.py
    ^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'main'
[2024-06-06T17:50:05.613+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:05.650+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.132 seconds
[2024-06-06T17:50:13.684+0000] {processor.py:161} INFO - Started process (PID=491) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:13.687+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:50:13.692+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:13.691+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:13.825+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:14.365+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:14.364+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:mi_dag_diario
[2024-06-06T17:50:14.391+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:14.390+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:mi_dag_diario
[2024-06-06T17:50:14.406+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:14.406+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:mi_dag_diario
[2024-06-06T17:50:14.408+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:14.408+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:50:14.439+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:14.439+0000] {dag.py:3118} INFO - Creating ORM DAG for mi_dag_diario
[2024-06-06T17:50:14.469+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:14.469+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:50:14.515+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.850 seconds
[2024-06-06T17:50:24.666+0000] {processor.py:161} INFO - Started process (PID=496) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:24.667+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:50:24.669+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:24.668+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:24.706+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:24.719+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:24.718+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:50:24.747+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:24.747+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:50:24.780+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.121 seconds
[2024-06-06T17:50:35.005+0000] {processor.py:161} INFO - Started process (PID=501) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:35.007+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:50:35.012+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:35.011+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:35.055+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:35.075+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:35.074+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:50:35.125+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:35.124+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:50:35.188+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.193 seconds
[2024-06-06T17:50:45.427+0000] {processor.py:161} INFO - Started process (PID=506) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:45.429+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:50:45.433+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:45.433+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:45.491+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:45.543+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:45.543+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:50:45.604+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:45.595+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:50:45.664+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.247 seconds
[2024-06-06T17:50:55.930+0000] {processor.py:161} INFO - Started process (PID=511) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:55.932+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:50:55.936+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:55.935+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:55.980+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:50:56.020+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:56.020+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:50:56.056+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:50:56.056+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:50:56.092+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.174 seconds
[2024-06-06T17:51:06.335+0000] {processor.py:161} INFO - Started process (PID=516) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:06.337+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:51:06.346+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:06.346+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:06.394+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:06.431+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:06.431+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:51:06.468+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:06.468+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:51:06.505+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.178 seconds
[2024-06-06T17:51:16.711+0000] {processor.py:161} INFO - Started process (PID=521) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:16.712+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:51:16.717+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:16.716+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:16.782+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:16.894+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:16.893+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:51:16.944+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:16.943+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:51:16.979+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.279 seconds
[2024-06-06T17:51:27.193+0000] {processor.py:161} INFO - Started process (PID=526) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:27.196+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:51:27.200+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:27.199+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:27.266+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:27.333+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:27.332+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:51:27.379+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:27.378+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:51:27.424+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.241 seconds
[2024-06-06T17:51:37.622+0000] {processor.py:161} INFO - Started process (PID=531) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:37.625+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:51:37.629+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:37.628+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:37.680+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:37.726+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:37.726+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:51:37.774+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:37.773+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:51:37.824+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.211 seconds
[2024-06-06T17:51:48.041+0000] {processor.py:161} INFO - Started process (PID=536) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:48.044+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:51:48.048+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:48.047+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:48.105+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:48.147+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:48.147+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:51:48.185+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:48.185+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:51:48.219+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.188 seconds
[2024-06-06T17:51:58.427+0000] {processor.py:161} INFO - Started process (PID=541) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:58.429+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:51:58.433+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:58.432+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:58.484+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:51:58.530+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:58.529+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:51:58.576+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:51:58.575+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:51:58.626+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.208 seconds
[2024-06-06T17:52:08.862+0000] {processor.py:161} INFO - Started process (PID=546) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:08.865+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:52:08.870+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:08.870+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:08.918+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:08.965+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:08.964+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:52:09.006+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:09.005+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:52:09.050+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.197 seconds
[2024-06-06T17:52:19.338+0000] {processor.py:161} INFO - Started process (PID=551) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:19.349+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:52:19.362+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:19.361+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:19.473+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:19.516+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:19.516+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:52:19.555+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:19.555+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:52:19.597+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.283 seconds
[2024-06-06T17:52:29.822+0000] {processor.py:161} INFO - Started process (PID=556) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:29.825+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:52:29.829+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:29.828+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:29.887+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:29.944+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:29.944+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:52:30.017+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:30.017+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:52:30.061+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.252 seconds
[2024-06-06T17:52:40.252+0000] {processor.py:161} INFO - Started process (PID=561) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:40.255+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:52:40.258+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:40.258+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:40.311+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:40.360+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:40.359+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:52:40.402+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:40.401+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:52:40.441+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.204 seconds
[2024-06-06T17:52:50.620+0000] {processor.py:161} INFO - Started process (PID=566) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:50.622+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:52:50.626+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:50.625+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:50.677+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:52:50.729+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:50.729+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:52:50.775+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:52:50.774+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:52:50.823+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.213 seconds
[2024-06-06T17:53:00.999+0000] {processor.py:161} INFO - Started process (PID=571) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:01.001+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:53:01.004+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:01.004+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:01.046+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:01.086+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:01.086+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:53:01.126+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:01.126+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:53:01.169+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.181 seconds
[2024-06-06T17:53:11.351+0000] {processor.py:161} INFO - Started process (PID=576) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:11.354+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:53:11.356+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:11.356+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:11.399+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:11.438+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:11.437+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:53:11.475+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:11.475+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:53:11.515+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.173 seconds
[2024-06-06T17:53:21.719+0000] {processor.py:161} INFO - Started process (PID=581) to work on /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:21.722+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2024-06-06T17:53:21.727+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:21.726+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:21.800+0000] {processor.py:840} INFO - DAG(s) 'mi_dag_diario' retrieved from /opt/airflow/dags/my_dag.py
[2024-06-06T17:53:21.929+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:21.929+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-06T17:53:21.983+0000] {logging_mixin.py:188} INFO - [2024-06-06T17:53:21.983+0000] {dag.py:3954} INFO - Setting next_dagrun for mi_dag_diario to 2024-06-05 00:00:00+00:00, run_after=2024-06-06 00:00:00+00:00
[2024-06-06T17:53:22.032+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/my_dag.py took 0.324 seconds
